### ISSUE TITLE
Translating FAQs and other documentation

### ISSUE THREAD

**Issue by Sappstal** [ShortDevelopment/Nearby-Sharing-Windows#99]
Translating the documentation would make it easier for users to set up the app, as some settings change depending on the language.

**Comment by ShortDevelopment** [comment]
That's right but I would rather use deeplinks into the settings app (like I did in the setup) instead of translating everything as it's a huge effort to update every time.
Could we maybe supply links for Google Translate in addition to deep links?
